catalogue/webapp/components/PhysicalItemDetails/PhysicalItemDetails.tsx

```diff
@@ -97,6 +113,7 @@ const PhysicalItemDetails: FunctionComponent<Props> = ({
       {accessNote && (
         <Row>
           <Box>
+            <DetailHeading>Item note</DetailHeading>
```

Comment:
Just spotted this in Zeplin too. I think I'd go for just `Note` here, as the item note is really the other, unlabelled note.